src/pdfkit.cs

```diff
@@ -508,9 +508,13 @@ interface PdfActionRemoteGoTo {
 	}
 
 	[iOS (11,0)]
+	[DisableDefaultCtor]
 	[BaseType (typeof (PdfAction), Name = "PDFActionResetForm")]
 	interface PdfActionResetForm {
-		//has a public Init ???
+		// - (instancetype)init NS_DESIGNATED_INITIALIZER;
+		[Export ("init")]
+		[DesignatedInitializer]
+		IntPtr Constructor ();
```

Comment:
@dalexsoto we added a type-level `[DesignatedDefaultCtor]` last year to cover that case